Mathlib/Data/Fintype/Order.lean

```diff
@@ -108,6 +107,12 @@ noncomputable abbrev toCompleteDistribLattice [DistribLattice α] [BoundedOrder
     simp_rw [Set.mem_toFinset]
     rfl
 
+-- Porting note: `convert` doesn't work as well as it used to.
```

Comment:
What is this porting note talking about here?

Reply:
Probably the remnant of a bad merge. Will remove